packages/mobile/src/app/saga.ts

```diff
@@ -124,6 +124,8 @@ export function* handleDeepLink(action: OpenDeepLink) {
       yield call(handlePaymentDeeplink, deepLink)
     } else if (rawParams.path.startsWith('/dappkit')) {
       handleDappkitDeepLink(deepLink)
+    } else if (rawParams.path.startsWith('/cashIn')) {
```

Comment:
Shouldn't this be more strict? We probably don't want to handle something like `/cashInSomething`, right?

Reply:
I don't see why not handling extra stuff. Maybe someone adds a query param like `/cashIn?someParam=true` and it would be fine to handle that imo.

I'm fine with having it be strict too though, changed it to an equality :)

Reply:
I assume this was only checking the path, and query parameters were in `rawParams.queryString` or something.

Reply:
I assumed that too at the beginning, but the `rawParams` has the following when opening the deep link `celo://wallet/cashIn?asd=qwe`:
```
{"auth": null, "hash": null, "host": "wallet", "hostname": "wallet", "href": "celo://wallet/cashIn?asd=qwe", "path": "/cashIn?asd=qwe", "pathname": "/cashIn", "port": null, "protocol": "celo:", "query": "asd=qwe", "search": "?asd=qwe", "slashes": true}
```
I guess we could use `pathname` instead of `path` but it doesn't really affect this usecase much, just something to think about in the future for other deep links :)